### PR TITLE
Add ;duel selfregister command enabled by env var

### PR DESCRIPTION
**Pull request by PotatoHashing**
Some servers prefer duelists register themselves instead of mods/admins registering them.

Environmental Variable set to false
image

Environmental variable set to true
image

Questions:
Do we want to allow duels by unidentified users?

### CODE REVIEW

**Comment by PotatoHashing** [comment]
Fixes #402

File: README.md [hunk 1]
```diff
@@ -79,7 +79,8 @@ Fill in appropriate variables in new "environment" file.
 #### Environment Variables
 
 **BOT_TOKEN**: the Discord Bot Token for your bot.  
-**LOGGING_COG_CHANNEL_ID**: the [Discord Channel ID](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) of a Discord Channel where you want error messages sent to.
+**LOGGING_COG_CHANNEL_ID**: the [Discord Channel ID](https://support.discord.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) of a Discord Channel where you want error messages sent to.
```
Comment (algmyr) [inline README.md:82]: Remove trailing whitespace.

**Review by algmyr — CHANGES_REQUESTED** [review]

**Comment by algmyr** [comment]
Other than that LGTM. I'll merge after the whitespace fix (unless other devs have opinions).

File: README.md (hunk 1 above)
Comment (PotatoHashing) [reply README.md]: Changed to bulleted list.

**Pull request merged**